packages/x-components/tests/e2e/cucumber/multiselect-filters.feature

```diff
@@ -13,18 +13,17 @@ Feature: MultiSelect filters component
     When  "<query>" is searched
     Then  related results are displayed
     When  waiting for search request intercept
-    And   filter number <filterNumber1> is selected in facet "<facetName>"
-    Then  selected filter is shown in the selected filters list
-    And   search request contains selected filter
+    And   filter "<filter1>" is clicked in facet "<facetName>"
+    Then  filter "<filter1>" is shown in the selected filters list
+    And   search request contains "<filter1>" filter
     When  waiting for search request intercept
-    And   filter number <filterNumber2> is selected in facet "<facetName>"
-  # TODO  https://searchbroker.atlassian.net/browse/EX-4866
-  # Then  selected filter is shown in the selected filters list
-  # And   search request contains selected filter
-  # TODO  https://searchbroker.atlassian.net/browse/EX-4866
-  # And   clear-filters button displays the number of selected filters
+    And   filter "<filter2>" is clicked in facet "<facetName>"
+    Then  filter "<filter1>" is shown in the selected filters list
+    And   filter "<filter2>" is shown in the selected filters list
+    And   search request contains "<filter1>" filter
+    And   search request contains "<filter2>" filter
+    And   clear-filters button should have 2 filters selected
```

Comment:
I'm not sure if I like this `2` here.

Reply:
Then think about it, and get back to this comment when you know it 🤣.

Reply:
I was thinking about using a variable!

Reply:
`DON'T USE MAGIC NUMBERS`

Reply:
This is not a magic number, as per wikipedia, magic numbers are:

> Unique values with unexplained meaning or multiple occurrences which could (preferably) be replaced with named constants

Here I'm sure this number was perfectly understood, the human readable sentence gave enough context about it. Anyway, I've changed it just for consistency. Even if I prefer to hardcode values in tests with only one example.